test/parallel/test-runner-tap-lexer.js

```diff
@@ -444,3 +444,16 @@ ok 1
     assert.strictEqual(tokens[index].value, token.value);
   });
 }
+
+// Test isLiteralSymbol method
+{
+  assert.strictEqual(isLiteralSymbol('A'), true);
+  assert.strictEqual(isLiteralSymbol('a'), true);
+  assert.strictEqual(isLiteralSymbol('-'), true);
+  assert.strictEqual(isLiteralSymbol('+'), true);
+  assert.strictEqual(isLiteralSymbol('0'), false);
+  assert.strictEqual(isLiteralSymbol('#'), false);
+  assert.strictEqual(isLiteralSymbol('أ'), true);
+  assert.strictEqual(isLiteralSymbol('ت'), true);
+  assert.strictEqual(isLiteralSymbol('ث'), true);
```

Comment:
Could you: 
1. group these into 2 groups: literals (`true`) and non-literals (`false`)?
2. add more samples of non-Latin characters? you can cherry-pick from this list [here](https://en.wikipedia.org/wiki/List_of_Unicode_characters).

Reply:
@manekinekko thank you so much i will fly here 🚀

Reply:
@manekinekko  would that be healthy

```js
{
  const literals = ['A', 'a', '-', '+', 'أ', 'ت', 'ث'];
  const nonLiterals = ['0', '#'];
  
  literals.forEach((literal) => {
  assert.strictEqual(isLiteralSymbol(literal), true);
  });
  
  nonLiterals.forEach((nonLiteral) => {
  assert.strictEqual(isLiteralSymbol(nonLiteral), false);
  });
}
```